lib/modules/manager/gradle/readme.md

```diff
@@ -1,2 +1,6 @@
 The `gradle` manager uses a custom parser written in JavaScript, similar to many others managers.
 It does not call `gradle` directly in order to extract a list of dependencies.
+
+### Updating lockfiles
+
+Updating lockfiles is done with `./gradlew :dependencies --wirte/update-locks` command. This command can output excessive text to the console. While running the command, the output to stdout is dropped when you run Renovate on most platfroms other than Windows.
```

Comment:
Use one line per sentence, please. 
See our developer docs for more infos.

Reply:
Thanks. TIL about one line per sentence. I ran `yarn doc-fix` but didn't get updates🤔 I manually fixed this.